src/main/java/org/fundacion/jala/converter/core/results/Result.java

```diff
@@ -0,0 +1,50 @@
+package org.fundacion.jala.converter.core.results;
+
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * *  This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Saul Caspa Miranda
+ */
+

```

Comment:
the package should be after the copyright

Reply:
And the copyright should start from line 1

Reply:
Done.